src/jit/lower.cpp

```diff
@@ -2862,8 +2862,8 @@ void Lowering::InsertPInvokeMethodProlog()
     firstBlockRange.InsertBefore(insertionPoint, LIR::SeqTree(comp, store));
     DISPTREERANGE(firstBlockRange, store);
 
-#ifndef _TARGET_X86_ // For x86, this step is done at the call site (due to stack pointer not being static in the
-                     // function).
+#if !defined(_TARGET_X86_) && !defined(_TARGET_ARM_)
+    // For x86 and arm32, this step is done at the call site (due to stack pointer not being static in the function).
 
```

Comment:
I've experimented to remove both these my changes about offsetOfCallSiteSP and offsetOfCalleeSavedFP, and it still works. But then I wonder how legacy backend works, because I can't see any analogous stores in it's output:
```
000006  F10D 0B60      add     r11, sp, 96
...
000030  4668           mov     r0, sp
000032  9006           str     r0, [sp+0x18]
000034  4658           mov     r0, r11
000036  9008           str     r0, [sp+0x20]
```